### DISCUSSION TITLE
Support for vouchers, coupons, giftcards and loyalty cards

### DISCUSSION THREAD

**Discussion by mzb2xeo** [dani-garcia/vaultwarden#4799] [Ideas]
Adding a item type named "Voucher" would be really useful instead of storing it in my cards or secure notes:

Ideally each would have:
-expiry notifications
-summary of the total outstanding value of your items
-QR or bar code for easy scanning during redemption
-transaction history

**Comment by BlackDex** [comment]
Not going to happen from this project. We do not maintain the clients and this needs to be supported by all clients, else it will not work. Features like this need to be supported by Bitwarden.

**Comment by l4rm4nd** [comment]
> Adding a item type named "Voucher" would be really useful instead of storing it in my cards or secure notes:
> 
> Ideally each would have:
> -expiry notifications
> -summary of the total outstanding value of your items
> -QR or bar code for easy scanning during redemption
> -transaction history
> 

Maybe check out VoucherVault instead.

https://github.com/l4rm4nd/VoucherVault
